src/pages/RepoPage/RepoPage.tsx

```diff
@@ -123,7 +124,7 @@ function Routes({
             ]}
             exact
           >
-            <BundlesTab />
+            <BundleOnboarding />
```

Comment:
Can you check if `jsOrTsPresent`? for some reason it causes infinite loop if it's ever hard coded in url.
maybe in https://github.com/codecov/gazebo/blob/66f3413100e3905d8f85c207a7901ad8d19f6933/src/pages/RepoPage/BundlesTab/BundleOnboarding/BundleOnboarding.tsx#L126

Reply:
What?

Reply:
What? I'm confused as to what you're talking about, this hard redirect will redirect you to `/:provider` and the code you've highlighted is protected by the check you're requesting.

Reply:
it's obviously not part of the change which is why i didn't see it :)